src/System.Linq.Expressions/src/System/Linq/Expressions/DynamicExpression.cs

```diff
@@ -1172,29 +1076,29 @@ public static DynamicExpression Dynamic(CallSiteBinder binder, Type returnType,
             return MakeDynamic(binder, returnType, args);
         }
 
-        private static DynamicExpression MakeDynamic(CallSiteBinder binder, Type returnType, ReadOnlyCollection<Expression> args)
+        private static DynamicExpression MakeDynamic(CallSiteBinder binder, Type returnType, ReadOnlyCollection<Expression> arguments)
```

Comment:
It helps to eliminate a magic string in favor of using `nameof(arguments)`, matching the name of the arguments collection passed to the (public) caller. In fact, as I write this, I noticed a place where we are using `args` incorrectly in a `paramName` argument for and `ArgumentException` so will fix that too in the next iteration.